Godeps/Godeps.json

```diff
@@ -499,47 +499,47 @@
 		},
 		{
 			"ImportPath": "github.com/containerd/containerd/api/services/containers/v1",
-			"Comment": "v1.0.0-beta.2-159-g27d450a",
+			"Comment": "v1.0.0-beta.2-159-g27d450a0",
```

Comment:
I don't think this line is supposed to have changed,
same thing was happening on my PR https://github.com/kubernetes/kubernetes/pull/63893#discussion_r189993079

Reply:
Although, the revision hash is the same so I think this change doesn't break anything

Reply:
Well, that's even different. This one is really just the same hash, for some reason godep though it would need more digit to avoid collision. That happens frmo time to time ...